Makefile

```
@@ -436,7 +433,7 @@ define testapp-set-replicas
 endef
 
 define demo-host
-	kubectl run -it --rm k8gbdemo --restart=Never --image=absaoss/k8gbdemocurl  \
+	kubectl run -it --rm k8gbdemo --restart=Never --image=absaoss/k8gbdemocurl --env="DEBUG=$(DEMO_DEBUG)" \
```

@ytsarev little suggestion to improve the readability of the image name (comparing to the rest of image naming in https://hub.docker.com/u/absaoss):
```suggestion
	kubectl run -it --rm k8gbdemo --restart=Never --image=absaoss/k8gb-demo-curl --env="DEBUG=$(DEMO_DEBUG)" \
```